src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -404,21 +404,21 @@ void HierRTLMP::hierRTLMacroPlacer()
 
   logger_->report(
       "Traversed logical hierarchy\n"
-      "\tNumber of std cell instances : {}\n"
-      "\tArea of std cell instances : {:.2f}\n"
-      "\tNumber of macros : {}\n"
-      "\tArea of macros : {:.2f}\n"
-      "\tArea of macros with halos : {:.2f}\n"
-      "\tTotal area : {:.2f}\n"
-      "\tDesign Utilization : {:.2f}\n"
+      "\tNumber of std cell instances: {}\n"
+      "\tArea of std cell instances: {:.2f}\n"
+      "\tNumber of macros: {}\n"
+      "\tArea of macros: {:.2f}\n"
+      "\tArea of macros with halos: {:.2f}\n"
+      "\tCore area: {:.2f}\n"
+      "\tDesign Utilization: {:.2f}\n"
       "\tCore Utilization: {:.2f}\n"
       "\tManufacturing Grid: {}\n",
       metrics_->getNumStdCell(),
       metrics_->getStdCellArea(),
       metrics_->getNumMacro(),
       metrics_->getMacroArea(),
       macro_with_halo_area,
-      metrics_->getStdCellArea() + metrics_->getMacroArea(),
+      core_area,
```

Comment:
Core area is a completely different thing.  Would total instance area be clearer?

Reply:
perhaps spell out the addition?

Reply:
What wording would make sense to you

Reply:
`Area of std cell instances + Area of macros:`

Reply:
Though, given the error message below, I would want to list core_area as well, it is relevant:

```C++
if (macro_with_halo_area + metrics_->getStdCellArea() > core_area) {
error
```

Reply:
That should be a separate entry then.

Reply:
Would 'Area of instances' suffice?

Reply:
In this case, I have to guess that a macro is considered an instance. 

Why not:

`Area of std cell instances + Area of macros:`

?

Much less than 80 chars...

Reply:
How could a macro not be an instance?  I don't care that strongly.

Reply:
I don't know how it could not.

You asked me what label would make the report easier to understand. I think `Area of std cell instances + Area of macros` is self documenting, whereas `Area of instance` leaves me with questions and I wouldn't have guessed that it is an addition of two other pieces of information listed.

I'm thinking that this report is going to be read on those(such as me) that are being trained in the art of backend tools, in which case a bit of spelling out is helpful.